site/content/en/docs/Reference/kustomize_cli.md

```diff
@@ -1,8 +1,77 @@
 ---
-title: "kustomize CLI"
-linkTitle: "kustomize CLI"
+title: "Command line tool (kustomize)"
+linkTitle: "Command line tool (kustomize)"
 weight: 2
 date: 2017-01-05
```

Comment:
I personally think that this page should go first, and then the API section should go next.
Right now CLI section's weight is 2 and API section's weight is 1.

```suggestion
weight: 0
date: 2017-01-05
```